.github/workflows/integrate.yaml

```diff
@@ -0,0 +1,37 @@
+# https://docs.github.com/en/actions
+
+name: "Integrate"
+
+on:
+  pull_request: null
+  push:
+    branches:
+      - "master"
+
+jobs:
+  tests:
+    name: "Tests"
+
+    runs-on: "ubuntu-18.04"
+
+    strategy:
+      matrix:
+        php-version:
+          - "7.2"
```

Comment:
Using PHP 7.2 here as laid out in `README.md`.

Reply:
Okay, let's go with PHP 7.2 then (although I'm not sure which version actually runs on php.net).